crud.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(skip: int, limit: int, db: Session):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def get_all_books(db: Session, skip: int = 0, limit: int = 5, author_id: int = None):
```

Comment:
The default values for `skip` and `limit` parameters should be consistent across all functions. Consider setting the default `limit` to 10 here to match the pagination defaults in other parts of the application.